index.html

```diff
@@ -16,45 +16,45 @@
                 <a class="desk-port" href="./index.html">Portfolio</a> 
                 <a class="desk-about" href="./index.html">About</a> 
                 <a class="desk-contact" href="./index.html">Contact</a>      
-                <img class="mailbox" src="./desktop-img/Icon - Mail.svg" alt="message icon">
```

Comment:
- [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast.

![image](https://user-images.githubusercontent.com/62290545/223443476-d7af5045-476c-4a45-abf0-d770efd394e9.png)
![image](https://user-images.githubusercontent.com/62290545/223443630-8987464a-45bc-4ad2-a35e-628aa1754578.png)

Reply:
@ShadyShawkat 

Please, I tried using the tool you suggested and this is the output I got.
![Capture2](https://user-images.githubusercontent.com/82339317/223469489-6a3504d3-ac3a-4a8d-afdb-39cf81bed1b5.PNG)